docs/kms/README.md

```diff
@@ -401,7 +401,7 @@ export MINIO_KMS_KES_ENDPOINT=https://localhost:7373
 export MINIO_KMS_KES_KEY_FILE=minio.key
 export MINIO_KMS_KES_CERT_FILE=minio.cert
 export MINIO_KMS_KES_KEY_NAME=minio-key-1
-export MINIO_KMS_KES_CAPATH=kes-tls.crt
```

Comment:
This is not correct. See: https://github.com/minio/minio/blob/master/cmd/crypto/config.go#L160

Reply:
this is `MINIO_KMS_KES_CA_PATH ` (line 186) , not `MINIO_KMS_VAULT_CAPATH ` :P

Reply:
Oh wait - that's a server code bug. Compare to https://github.com/minio/minio/blob/master/cmd/crypto/config.go#L186
/cc @harshavardhana